com.exok.Celeste64.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Name=Celeste 64
+Exec=/app/bin/celeste64/bin/Release/net8.0/linux-x64/Celeste64
```

Comment:
exec should be the name of the executable (not full path) that in "PATH" (/app/bin). You can make a symlink there for that purpose

Reply:
do you mean here it should be only `Celeste64`? I could change the build steps to get the contents of `linux-x64` in the root of `/app/bin` in this case probably. Would that work?